{{cookiecutter.project_slug}}/clients/vue3/package.json

```diff
@@ -17,13 +17,16 @@
     "@thinknimble/tn-models": "1.2.1",
     "@thinknimble/tn-utils": "1.6.0",
     "@thinknimble/tn-validators": "^1.1.1",
+    "autoprefixer": "^9.8.8",
+    "axios": "^0.22.0",
     "core-js": "^3.6.5",
+    "js-cookie": "^3.0.1",
+    "postcss": "^7.0.39",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.2.17",
```

Comment:
**Info:** Package versions referenced via this GitHub discussion regarding PostCSS 8:

https://github.com/tailwindlabs/tailwindcss/discussions/2854